doc/CONFIGURATION.md

```diff
@@ -4,7 +4,6 @@
 - [Modules](#modules)
   - [Transport](#transport)
   - [Stream Multiplexing](#stream-multiplexing)
-    - [Muxer Selection](#muxer-selection)
```

Comment:
Why has this been removed?

Reply:
Right, looks like the header was removed, https://github.com/libp2p/js-libp2p/pull/1848 restores it